src/pybind/mgr/dashboard/frontend/protractor.conf.js

```diff
@@ -34,8 +34,8 @@ const config = {
   },
   params: {
     login: {
-      user: 'admin',
-      password: 'admin'
+      user: process.env.E2E_USER || 'admin',
```

Comment:
What about being more precise in naming the env variable? I know it is improbable, but mayge we need a E2E_USER someday.
```suggestion
      user: process.env.E2E_LOGIN_USER || 'admin',
```

Reply:
Done.